docs/operate/get-started/other-hardware/_index.md

```diff
@@ -559,26 +559,63 @@ Make sure to physically connect your sensor to your machine's computer to prepar
 **1. Prepare to run your module**
 
 {{< tabs >}}
-{{% tab name="Python: pyinstaller (recommended)" %}}
-{{% alert title="Note" color="note" %}}
-To follow these PyInstaller packaging steps, you must have enabled cloud build when moving through the module generator prompts.
-If you did not, you will need to manually create a <file>build.sh</file> entrypoint script.
-{{% /alert %}}
+{{% tab name="Python: PyInstaller (recommended)" %}}
 
-From within the <file>hello-world</file> directory, create a virtual Python environment with the necessary packages and then build an executable by running the setup and build scripts:
+If you enabled cloud build, use these steps.
 
-```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
-sh setup.sh
-sh build.sh
-```
+1. Create a <file>reload.sh</file> script in your module directory.
+   You'll use this for local testing and can delete it before you upload your module.
+   Paste the following contents into it and save the file:
 
-This environment is where the local module will run.
-`viam-server` does not need to run inside this environment.
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   #!/usr/bin/env bash
+
+    # bash safe mode. look at `set --help` to see what these are doing
+    set -euxo pipefail
+
+    cd $(dirname $0)
+    MODULE_DIR=$(dirname $0)
+    VIRTUAL_ENV=$MODULE_DIR/venv
+    PYTHON=$VIRTUAL_ENV/bin/python
+    ./setup.sh
+
+    # Be sure to use `exec` so that termination signals reach the python process,
+    # or handle forwarding termination signals manually
+    exec $PYTHON src/main.py $@
+   ```
+
+1. Make your reload script executable by running the following command in your module directory:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   chmod 755 reload.sh
+   ```
+
+1. Create a virtual Python environment with the necessary packages by running the setup file from within the module directory:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   sh setup.sh
+   ```
+
+1. Edit your <file>meta.json</file>, replacing the `"entrypoint"`, `"build"`, and `"path"` fields as follows:
+
+   ```json {class="line-numbers linkable-line-numbers" data-start="13" data-line="1, 4, 6" }
+     "entrypoint": "reload.sh",
+     "first_run": "",
+     "build": {
+       "build": "rm -f module.tar.gz && tar czf module.tar.gz requirements.txt src/*.py meta.json setup.sh reload.sh",
```

Comment:
This is fine for now, don't change anything. Im my updates for the generator to use hotreloading by default, I will probably create a makefile or sh script. Will let you know how that goes